### ISSUE TITLE
Kinyarwanda word for 'keywords' is too long. Getting cut off behind search box on rw/list/successful

### ISSUE THREAD

**Issue by StephenAbbott** [mysociety/sobanukirwa-theme#73]
The Kinyarwanda word for keywords is 'Amagamboshingiro' which is being cut off behind the search box when viewing the https://sobanukirwa.rw/rw/list/successful page on wide desktop browsers. Works fine on mobile. Any way to fix this?

**Comment by garethrees** [comment]
**Note to self**

Fix:

```css
#filter_requests_form label.title, #search_form label.title, #filter_form label.title {
display: inline-block;
float: none;
/* width: 110px; */
```

**Comment by garethrees** [comment]
Fixed in 6f374b8
